### ISSUE TITLE
[1.2] SpaceAroundOperator false-positive

### ISSUE THREAD

**Issue by boris-petrov** [CodeNarc/CodeNarc#348]
This is a regression from 1.1:

```groovy
class DriverLoader {
	static loadDriver(driverName) {
		def driver = Class.forName(driverName)
		GroovyObject.classLoader.addURL(driver.protectionDomain.codeSource.location)
	}
}
```

This gives a `SpaceAroundOperator` warning for `def driver = Class.forName(driverName)`.

**Comment by boris-petrov** [comment]
Actually no, the problem occurs whenever a `SuppressWarnings` annotation is used on a particular line:

```groovy
@SuppressWarnings('UnnecessarySubstring')
def relativePath = filePath.substring(path.length())
```

```groovy
@SuppressWarnings('ClassForName')
def driver = Class.forName(driverName)
```

**Comment by nre-ableton** [comment]
I'm stuck with CodeNarc 1.1 because of this bug, since we require `@SuppressWarnings('VariableTypeRequired')` for any variable declared with `def`, which triggers this error. A fix would be greatly appreciated.

**Comment by chrismair** [comment]
Available in CodeNarc 1.2.1
